### ISSUE TITLE
REST API: Can not list copied uploads

### ISSUE THREAD

**Issue by GMishx** [fossology/fossology#1858]
### Description

If an upload created by other user is copied into folder of other user, REST API can not list the upload as it filters the result by user id as well.

#### How to reproduce

1. From User 1, upload a package in a folder.
2. From Admin user, copy the upload to another folder accessible by User 2.
3. From User 2, query for uploads, query for copied upload, the result will not contain the concerned upload.

### Versions

* Last commit id on master: dafcc14f89a7691292df4bbe197bb991dd41d681

#### Apache logs

```
PHP Notice:  Undefined offset: 0 in /usr/share/fossology/www/ui/api/Controllers/UploadController.php on line 88
```

**Comment by b-thebest** [comment]
Hi, I am new to open source, Can I work on this issue?

**Comment by GMishx** [comment]
Hello @b-thebest , thank you for your interest. I think @OmarAbdelSamea is also working on the same issue.

**Comment by b-thebest** [comment]
Okay, no worries

**Comment by GMishx** [comment]
Fixed in #1764
